virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="virt-configuring-shared-volumes-for-vms"]
+= Configuring shared volumes for virtual machines
+include::_attributes/common-attributes.adoc[]
+:context: virt-configuring-shared-volumes-for-vms
+
+toc::[]
+
+
+You can configure shared disks to allow multiple virtual machines (VMs) to share the same underlying storage. A shared disk's volume must be block mode.
+
+You configure disk sharing by exposing the storage in either of the following ways:
+
+* As an ordinary virtual machine disk.
+* As a logical unit number (LUN) device with an iSCSi connection and raw device mapping, as required for Windows Failover Clustering for shared volumes. For more information, see link:https://www.qemu.org/docs/master/interop/pr-helper.html[Persistent reservation helper protocol] and link:https://learn.microsoft.com/en-us/windows-server/failover-clustering/failover-clustering-overview[Failover Clustering in Windows Server and Azure Stack HCI].
```

Comment:
You might want to put these links in an "Additional resources" section because they are not directly related to the bullet text.

Reply:
Done